packages/api-openfin/src/window.js

```diff
@@ -10,23 +10,29 @@ class Window {
       let newWindow;
       const handleError = (error) => console.error('Error creating window: ' + error);
 
-      if (features && features.child) {
-        newWindow = new fin.desktop.Window({
+      const windowOptions = features || {};
+      if (windowOptions.transparent) {
+        // OpenFin needs opacity between 1 (not transparent) and 0 (fully transparent)
+        windowOptions.opacity = features.transparent === true ? 0 : 1;
+      }
+
+      if (windowOptions.child) {
+        Object.assign(windowOptions, {
```

Comment:
I'm not a fan of overriding `windowOptions`. I'd assign a new `const` and use that instead